mod_wasm/modules/wasm/mod_wasm.h

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright 2022 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// mod_wasm version
+#define MOD_WASM_VERSION_MAJOR 0
+#define MOD_WASM_VERSION_MINOR 8
+#define MOD_WASM_VERSION_PATCH 0
```

Comment:
I saw we have a different file with pretty similar information: `wasm_runtime/include/version.h`. What's the difference between both and why are required? I also noticed that `version.h` is automatically generated by the `build.rs` file.

Reply:
For the moment, we need two different version numbers, one for `mod_wasm.so`, and the other for `libwasm_runtime.so`.

Eventually, if both libs stay together, we can merge versions 👍